src/GenQPSolver.h

```diff
@@ -112,6 +119,16 @@ class TASKS_DLLAPI GenQPSolver
 		const std::vector<GenInequality*>& genInEqConstr,
 		const std::vector<Bound*>& boundConstr,
 		std::ostream& out) const = 0;
+protected:
+	/** Correspondance between full variable indices and reduced variables */
```

Comment:
Note that in all those comments it should really be "Correspondence" or maybe "Matching".